### PR TITLE
Update pydantic rnaf tomte

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b update-pydantic-rnaf-tomte
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/models/nf_analysis.py [hunk 1]
```diff
@@ -39,20 +39,22 @@ class NextflowSampleSheetEntry(BaseModel):
     """
 
     name: str
-    fastq_forward_read_paths: conlist(Path, min_items=1)
-    fastq_reverse_read_paths: conlist(Path, min_items=1)
+    fastq_forward_read_paths: conlist(Path, min_length=1)
+    fastq_reverse_read_paths: conlist(Path, min_length=1)
```
Comment (diitaz93) [inline cg/models/nf_analysis.py:43]: I think it should be `min_items`

**Review by diitaz93 — APPROVED** [review]
👍

File: cg/models/nf_analysis.py (hunk 1 above)
Comment (henrikstranneheim) [reply cg/models/nf_analysis.py:43]: I think it is correct for Pydantic v2.

**Comment by henrikstranneheim** [comment]
###  cg start workflows, 👍 
```
nrik.stranneheim@hasta:~] [S_base] $ cg workflow tomte start wealthyglider --dry-run
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Starting analysis for wealthyglider
Case wealthyglider exists in Status DB
Dry run: Config files will not be written
Case wealthyglider exists in Status DB
Samples linked to case wealthyglider: 1
Sample ACC15717A5 not found in LIMS: 404: Sample not found: ACC15717A5
Creating gene panel file
Running command /home/proj/stage/bin/miniconda3/envs/S_scout/bin/scout --config /home/proj/stage/servers/config/hasta.scilifelab.se/scout-stage.yaml export panel --bed PANELAPP-GREEN OMIM-AUTO --build GRCh38
Dry run: process call will not be executed!!
Case wealthyglider exists in Status DB
Running analysis for wealthyglider
Workflow will be executed using Tower
Running command /home/proj/stage/bin/tw launch --work-dir /home/proj/stage/analysis/cases/wealthyglider/work --profile singularity --params-file /home/proj/stage/analysis/cases/wealthyglider/wealthyglider_params_file.yaml --config /home/proj/stage/analysis/cases/wealthyglider/wealthyglider_nextflow_config.json --name wealthyglider --revision 2.0.1 --compute-env hasta-normal gms_tomte
Dry run: process call will not be executed!!

Dry-run: Action running would be set for case wealthyglider
[henrik.stranneheim@hasta:~] [S_base] $ cg workflow rnafusion  start  cuddlyhen --dry-run
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Starting analysis for cuddlyhen
Case cuddlyhen exists in Status DB
Illumina run check is not applicable - the case is either down sampled or external.
Check if pending compression file exists
/home/proj/stage/housekeeper-bundles/surelyrightserval/2023-02-06/HCKLTDSX2_ACC8869A1_S6_L001.crunchy.pending.txt does not exist
Compression/decompression is not running
Check if FASTQ pair exists
Check if pending compression file exists
/home/proj/stage/housekeeper-bundles/surelyrightserval/2023-02-06/HCKLTDSX2_ACC8869A1_S6_L001.crunchy.pending.txt does not exist
Compression/decompression is not running
Both /home/proj/stage/housekeeper-bundles/surelyrightserval/2023-02-06/HCKLTDSX2_ACC8869A1_S6_L001_R1_001.fastq.gz and /home/proj/stage/housekeeper-bundles/surelyrightserval/2023-02-06/HCKLTDSX2_ACC8869A1_S6_L001_R2_001.fastq.gz from sample surelyrightserval are already in Housekeeper
Check if pending compression file exists
/home/proj/stage/housekeeper-bundles/surelyrightserval/2023-02-06/HCKLTDSX2_ACC8869A1_S6_L001.crunchy.pending.txt does not exist
Compression/decompression is not running
Check if FASTQ pair exists
Check if pending compression file exists
/home/proj/stage/housekeeper-bundles/surelyrightserval/2023-02-06/HCKLTDSX2_ACC8869A1_S6_L001.crunchy.pending.txt does not exist
Compression/decompression is not running
Dry run: Config files will not be written
Case cuddlyhen exists in Status DB
Samples linked to case cuddlyhen: 1
Case cuddlyhen exists in Status DB
Running analysis for cuddlyhen
Workflow will be executed using Tower
Running command /home/proj/stage/bin/tw launch --work-dir /home/proj/stage/rnafusion/cases/cuddlyhen/work --profile singularity,hasta --params-file /home/proj/stage/rnafusion/cases/cuddlyhen/cuddlyhen_params_file.yaml --config /home/proj/stage/rnafusion/cases/cuddlyhen/cuddlyhen_nextflow_config.json --name cuddlyhen --revision 3.0.1 --compute-env hasta-normal nfcore_rnafusion
Dry run: process call will not be executed!!

Dry-run: Action running would be set for case cuddlyhen

**Comment by henrikstranneheim** [comment]
Deployed to prod
```
done.
Log deploy... done.
cg, version 64.2.2

**Pull request merged**